### PR TITLE
Virtual files: don't check for new big folders

### PR DESCRIPTION
**Pull request by ogoffart**
https://github.com/owncloud/client/issues/6724#issuecomment-417368475

### CODE REVIEW

**Comment by michaelstingl** [comment]
@ogoffart same for mounted storages?

**Comment by ogoffart** [comment]
> same for mounted storages?

Yes.

Since the "choose what to sync" would be hidden (https://github.com/owncloud/client/pull/6753 ) , these feature makes no sense as there are no way to select files.

File: src/libsync/discoveryphase.cpp [hunk 1]
```diff
@@ -90,7 +90,7 @@ int DiscoveryJob::isInSelectiveSyncBlackListCallback(void *data, const QByteArra
 
 bool DiscoveryJob::checkSelectiveSyncNewFolder(const QString &path, RemotePermissions remotePerm)
 {
-    if (_syncOptions._confirmExternalStorage
+    if (_syncOptions._confirmExternalStorage && !_syncOptions._moveFilesToTrash
```
Comment (ckamm) [inline src/libsync/discoveryphase.cpp]: `_moveFilesToTrash`? I don't get it

Comment (ogoffart) [reply src/libsync/discoveryphase.cpp]: That obviously was meant to be _newFilesAreVirtual
I guess i selected the wrong entry in auto-complete by mistake.

**Pull request merged**